docs/source/user_guide/faq.rst

```diff
@@ -30,6 +30,33 @@ Should I use PySpark's DataFrame API or Koalas?
 If you are already familiar with pandas and want to leverage Spark for big data, we recommend
 using Koalas. If you are learning Spark from ground up, we recommend you start with PySpark's API.
 
+Can I use Koalas and PySpark in Juypter?
+------------------------------------------
+
+If you have installed Koalas and PySpark locally on your machine and wish
```

Comment:
I'm not pretty sure, but if i already **have installed Koalas and PySpark**, couldn't i just use them on Jupyter without any additional settings like the below?

![스크린샷 2020-01-29 오전 10 36 15](https://user-images.githubusercontent.com/44108233/73322970-44c5fc00-4289-11ea-9e30-7d7509a6af6b.png)

although i never think it is not useful, but as @HyukjinKwon said it seems more like instructions for PySpark, so it's a bit questionable whether this should be in Koalas FQA. 🤔

Reply:
Yes, I tried and it worked fine, after installing Koalas and PySpark, without setting the environment variables. Rather than place it in FAQ, under Getting Started or a  separate section "Collaboration with External Tools." 

Let me know and I can close this PR. thanks!

Reply:
Let's leave this close for now.